dcos/dcos-ui
# DCOS-57500: [UI] Add filter hiding community packages to catalog page

Add a filter to the pacakges page that
allows the user to filter for certified, community
and all packages with certified as default.

Closes https://jira.mesosphere.com/browse/DCOS-57500

## Testing
1. Go to Catalog tab.
2. Verify that the new filters are shown.
3. Verify that clicking them works as expected.
4. Verify that after using the input filter, they work as expected (changing the badge nubmers, etc.).
5. Verify that the added tests make sense.

## Trade-offs
See my comments.

## Dependencies
None.

## Screenshots
![Screenshot from 2019-08-20 14-03-50](https://user-images.githubusercontent.com/40791275/63346766-62984800-c35e-11e9-9481-f7db672006b3.png)
![Screenshot from 2019-08-20 14-05-35](https://user-images.githubusercontent.com/40791275/63346774-64faa200-c35e-11e9-9c3e-274e663c5343.png)
